cmd/xl-v1-multipart.go

```diff
@@ -741,13 +739,15 @@ func (xl xlObjects) CompleteMultipartUpload(ctx context.Context, bucket string,
 		partsMetadata[index].Parts = xlMeta.Parts
 	}
 
+	tempXLMetaPath := mustGetUUID()
+
```

Comment:
we should write the delete code here using xl.deleteObject() and remove it from commitXLMeta()

Reply:
> we should write the delete code here using xl.deleteObject() and remove it from commitXLMeta()

Fixed